qiita_db/analysis.py

```diff
@@ -282,7 +282,7 @@ def shared_with(self):
 
     @property
     def all_associated_filepath_ids(self):
-        """Get all associated filepath_ids EXCEPT job results filepaths
```

Comment:
I understand that there are less object instantiations this way, but do we want the analysis object interacting with jobs like this? Not sure how much blending of the two objects we want to allow.